package.yaml

```diff
@@ -76,6 +76,7 @@ library:
     - vty-crossplatform
     - wai-extra
     - warp
+    - wreq == 0.5.4.3
```

Comment:
do we need the strict pin here?

Reply:
I'm not sure. Using no pin will fail, so any suggestion is appreciated.